frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -34,11 +36,14 @@ function EmbeddingOption({
   children,
   icon,
 }: EmbeddingOptionProps) {
+  const titleId = useUniqueId();
   return (
-    <StyledCard compact>
+    <StyledCard compact role="article" aria-labelledby={titleId}>
```

Comment:
To make some tests work, because we were relying on fake id e.g. `data-testid="-static-embedding-setting"`. This makes it more accessible as well.